examples/bicluster/plot_spectral_coclustering.py

```diff
@@ -23,18 +23,27 @@
 from matplotlib import pyplot as plt
 
 from sklearn.datasets import make_biclusters
-from sklearn.datasets import _samples_generator as sg
 from sklearn.cluster import SpectralCoclustering
 from sklearn.metrics import consensus_score
 
+
+def shuffle(data, random_state=None):
```

Comment:
can't we use `sklearn.utils.shuffle` instead?

Reply:
Do not think `sklearn.utils.shuffle` can be used since this "shuffles" in both dimensions. 

I removed the `shuffle` method and did the shuffle in a few lines of code directly.